artistools/lightcurve.py

```diff
@@ -13,16 +13,25 @@
 import pandas as pd
 from astropy import constants as const
 from astropy import units as u
+import statistics as stat
```

Comment:
Looks like this is only used to calculate a standard deviation. use numpy.std instead to save an import (which add lag to the startup time)